kratos/python/add_vector_to_python.cpp

```diff
@@ -24,6 +24,7 @@
 #include "includes/ublas_complex_interface.h"
 #include "containers/array_1d.h"
 #include "python/add_vector_to_python.h"
+#include "utilities/quaternion.h"
```

Comment:
can you please put this in a separate file? i.e. `add_quaternion_to_python`
I often have problems finding the py-exposure of some things, because they are in files where one would not expect them

I see that there is little code for now, but usually those things grow over time

Reply:
done!